### PR TITLE
deploying my page

### PR DESCRIPTION
**Pull request by VICTOR-CHUKWU**
setting up my Github page for portfolio

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
```
Comment (carlos-ssh) [inline README.md:2]: - Provide a link where the website is deployed using GitHub pages.
- Updated The README file that includes a link to the online version. 
- Update the image of your README file.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @VICTOR-CHUKWU , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (VICTOR-CHUKWU) [reply README.md:2]: I really do not understand, my README file already has my online link and the image is up to date. about link where my website is deployed, I do not understand. thanks

**Review by AshakaE — APPROVED** [review]
### Hi @VICTOR-CHUKWU  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the required changes 👍 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**